src/forge/controller/launcher.py

```
@@ -32,28 +33,14 @@
     from torchx.specs import AppState
     from torchx.specs.fb.component_helpers import Packages
 except ImportError as e:
-    print(f"Warning: Monarch meta/fb inetrnal imports failed: {e}")
-    print("Monarch functionality will be limited")
+    # This means there is an error with MAST
```

should it silently fail?

---

I've just added in a block that says "if imports failed and you're trying to use MAST, print an error messaging say imports failed and that you should check your build was correct"